### ISSUE TITLE
Conflict with Rainlab Translate Plugin when version 2 installed

### ISSUE THREAD

**Issue by cyril-design** [OFFLINE-GmbH/oc-mall-plugin#963]
I cannot install Mall plugin : **Failed to update composer dependencies**
```
offline/oc-mall-plugin[v2.0.0, ..., v2.0.10] require rainlab/translate-plugin ^1.9.0 -> found rainlab/translate-plugin[v1.9.0, ..., 1.x-dev] but it conflicts with your root composer.json require (^2.0).
```

Thanks for your help !

**Comment by tobias-kuendig** [comment]
We currently do not support version 2.0 of the RainLab.Translate plugin. Or rather, it is completely untested, but might work.

You can change mall's translate plugin requirement to ^2.0 in the composer.json and check if it works.

**Comment by cyril-design** [comment]
Thanks for your reply Tobias.
I preferred to downgrade Rainlab Plugin to 1.9 ; I tested a bit the compatibility, and I noticed some strange behaviors (which I will try to trace).
Anyway I don't want to use the multisite feature, so...

**Comment by c2webstudio** [comment]
There are quick fixes for this issue but then it will break when used with versions prior v2.0 of Translate plugin.

**Comment by SamBrishes** [comment]
This issue has been fixed and released with v3.2.0 / v3.2.1.
